ledger/src/blockstore.rs

```diff
@@ -4309,7 +4308,7 @@ pub mod tests {
     use {
         super::*,
         crate::{
-            blockstore_options::BlockstoreRocksFifoOptions,
+            blockstore_options::{BlockstoreRocksFifoOptions, ShredStorageType},
```

Comment:
nit: `ShredStorageType` shouldn't be needed since we have a `use super::*` above, and we already pull `ShredStorageType` in for non-tests